### PR TITLE
[OSDEV-1145] Error message appearing as red dot with no context

### PR DESCRIPTION
**Pull request by Innavin369**
[OSDEV-1145] (https://opensupplyhub.atlassian.net/browse/OSDEV-1145) Error message appearing as a red dot with no context.

* Removed const variable with components and error messages, tests
* Added function to parse tag
* Added function to check error string contains a tag

The task which was implemented [OSDEV-1105](https://opensupplyhub.atlassian.net/browse/OSDEV-1105)

[OSDEV-1145]: https://opensupplyhub.atlassian.net/browse/OSDEV-1145?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[OSDEV-1105]: https://opensupplyhub.atlassian.net/browse/OSDEV-1105?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/react/src/components/ContributeForm.jsx [hunk 1]
```diff
@@ -94,6 +94,30 @@ class ContributeForm extends Component {
 
     handleUploadList = () => this.props.uploadList(this.fileInput);
 
+    containsTag = (content, tagName) => {
+        const tagRegex = new RegExp(`<${tagName}>.*?<\\/${tagName}>`, 'i');
+        return tagRegex.test(content);
+    };
+
+    parseContent = content => {
+        if (!this.containsTag(content, 'strong')) {
```
Comment (VadimKovalenkoSNF) [inline src/react/src/components/ContributeForm.jsx:103]: Why you need to check tags here? Is this come from the backend?

**Review by VadimKovalenkoSNF — CHANGES_REQUESTED** [review]
Left concern about parsing HTML in React component.

File: src/react/src/components/ContributeForm.jsx (hunk 1 above)
Comment (Innavin369) [reply src/react/src/components/ContributeForm.jsx:103]: No. it's not. I check if a string contains tags not to proceed with function logic if it's no sens. Moved logic to separate function.

Comment (VadimKovalenkoSNF) [reply src/react/src/components/ContributeForm.jsx:103]: Why string can contain tags?

File: src/react/src/util/constants.jsx [hunk 2]
```diff
@@ -1272,18 +1272,3 @@ export const facilityClaimStepsNames = Object.freeze({
     SUPPORT_DOC: 'Support Documentation',
     ADDITIONAL_DATA: 'Additional Data',
 });
-
-export const componentsWithErrorMessage = Object.freeze({
```
Comment (vladsha-dev) [inline src/react/src/util/constants.jsx:1276]: Why did you decide to delete `componentsWithErrorMessage` ?

**Review by vladsha-dev — CHANGES_REQUESTED** [review]

File: src/react/src/components/ContributeForm.jsx (hunk 1 above)
Comment (Innavin369) [reply src/react/src/components/ContributeForm.jsx:103]: I added the tag to style List Name bold. It's requirements from the client.

File: src/react/src/util/constants.jsx (hunk 2 above)
Comment (Innavin369) [reply src/react/src/util/constants.jsx:1276]: I added this componentsWithErrorMessage in the previous PR. But the rest of errors messages(from BE) stayed as string and that became the reason of the bug.

File: src/react/src/components/ContributeForm.jsx (hunk 1 above)
Comment (VadimKovalenkoSNF) [reply src/react/src/components/ContributeForm.jsx:103]: Where can I see this requirement? OSDEV-1145 doesn't contain any links to origin ticket.

Comment (Innavin369) [reply src/react/src/components/ContributeForm.jsx:103]: My bad https://opensupplyhub.atlassian.net/browse/OSDEV-1105. Added to PR description

Comment (VadimKovalenkoSNF) [reply src/react/src/components/ContributeForm.jsx:103]: Ok, I see it now. Still, attempting to parse HTML using regular expression [considering as antipattern](https://stackoverflow.com/a/1732454). Please, find another solution that solve this issue and keep the codebase consistent at the same time.

File: src/react/src/util/constants.jsx (hunk 2 above)
Comment (vladsha-dev) [reply src/react/src/util/constants.jsx:1276]: Then, you can make each error message just a string.

Comment (vladsha-dev) [reply src/react/src/util/constants.jsx:1276]: Then you can make each error message just a string.

**Pull request closed without merging**